clients/ctl/admin-ui/src/features/taxonomy/hooks.tsx

```diff
@@ -160,15 +169,22 @@ export const useDataUse = (): TaxonomyHookData<DataUse> => {
       />
       <CustomCreatableMultiSelect
         name="recipients"
-        label="Recipients"
+        label={labels.recipient}
         options={[]}
         size="sm"
       />
-      {/* TODO: legitimate interest: boolean field */}
-      <CustomTextInput
-        name="legitimate_interest_impact_assessment"
-        label={labels.legitimate_interest_impact_assessment}
+      <CustomRadioGroup
+        name="legitimate_interest"
+        label={labels.legitimate_interest}
+        options={YesNoOptions}
       />
+      {formValues.legitimate_interest &&
+      formValues.legitimate_interest.toString() === "true" ? (
```

Comment:
Nit, but does this work the same if you made it one line like `formValues.legitimate_interest?.toString() === "true" ? (`

Reply:
yeah I think you're right!